pkg/process/config/tracer_config.go

```diff
@@ -75,6 +75,10 @@ func SysProbeConfigFromConfig(cfg *AgentConfig) *ebpf.Config {
 		tracerConfig.OffsetGuessThreshold = th
 	}
 
+	if cfg.Windows.EnableMonotonicCount {
+		tracerConfig.EnableMonotonicCount = true
+	}
```

Comment:
Can this be written as `tracerConfig.EnableMonotonicCount = cfg.Windows.EnableMonotonicCount`